onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -1209,6 +1209,17 @@ Status TensorrtExecutionProvider::OnRunEnd(bool sync_stream) {
   return Status::OK();
 }
 
+nvinfer1::IBuilder& TensorrtExecutionProvider::GetBuilder() const {
```

Comment:
should we document this isn't thread-safe and can only be called during single thread init phases ? (GetCapability() and Compile()) ?

Reply:
Yes, we should document this even though it doesn't make sense to have multiple threads running the inference session initialization.